pySDC/implementations/sweeper_classes/Multistep.py

```diff
@@ -0,0 +1,219 @@
+from pySDC.core.Sweeper import sweeper, _Pars
+
+
+class Cache(object):
```

Comment:
That's cool! I like the class, pretty good idea! So you wrote a class having the values from previous steps and can update them using the `update` method? I was thinking way too complicated when I was implementing that..

Reply:
Yes, but this is actually a bit too simple. I know both of us want to restart time steps and that is not really supported with this simple cache. But we can add that later. For instance, we can keep one extra solution in the cache and do a reverse `update` if we restart a step. 
We should sit down while we're here at Hamburg at some point so you can show me what you already did and we can merge our efforts.